lib/commands/fastboot-build.js

```diff
@@ -20,7 +20,8 @@ module.exports = {
       project: this.project
     });
 
-    deprecate("Use of ember fastboot:build is deprecated. Please use ember build instead.");
+    deprecate('ember fastboot:build will be deprecated and removed after FastBoot 1.0 is released.');
```

Comment:
Yeah but I just wanted to update the deprecation to say we will be dropping this command too.